### ISSUE TITLE
No qualities from Radarr when approving a request

### ISSUE THREAD

**Issue by CBers** [tidusjar/Ombi#1049]
#### Ombi Version:

V2.1.1347

#### Update Branch:

development

#### Operating System:

Windows 7

#### Mono Version (only if your not on windows)

(Place text here)

#### Applicable Logs (from `/logs/` directory or the Admin page):

```

(Logs go here. Don't remove the ``` tags for showing your logs correctly. Please make sure you remove any personal information from the logs)

```

#### Problem Description:

No option to selct the quality of a movie when approving the request.

I use Radarr and I do have quality profiles set up.

works OK with Sonarr.

#### Reproduction Steps:

Please include any steps to reproduce the issue, this the request that is causing the problem etc.

**Comment by tidusjar** [comment]
Are they appearing in the settings?

**Comment by CBers** [comment]
Yes. 

Also, just added a movie which was already in Radarr, but there was no feedback when adding the request. 

In the Admin account, there was an error adding the movie to Radarr, when approving the request:

2017/02/04 12:27:06.137 Ombi.Core.MovieSender Error: This series has already been added 
2017/02/04 12:27:06.303 Ombi.UI.Modules.ApprovalModule Warn: We approved movie: Rogue One: A Star Wars Story but could not add it to CouchPotato/Watcher because it has not been setup 

Need to change the message to include Radarr, or streamline the message.
